File: maxhanna.Server/Controllers/DataContracts/StoryComment.cs
namespace maxhanna.Server.Controllers.DataContracts
{
    public class StoryComment
    {
        public int Id { get; set; }
        public int StoryId { get; set; }
        public User? User { get; set; } 
        public string? Text { get; set; }
        public int Upvotes { get; set; }
        public int Downvotes { get; set; }
        public DateTime Date { get; set; }
    }
}

File: maxhanna.client/src/services/datacontracts/story-comment.ts
import { User } from "./user";

export class StoryComment {
  id: number | undefined;
  storyId: number | undefined;
  user: User | undefined; 
  text: string | undefined;
  upvotes: number | undefined;
  downvotes: number | undefined;
  date: Date | undefined;
}
